src/main/java/com/photolocos/enterprise/controller/PhotoLocosController.java

```diff
@@ -15,6 +15,7 @@
 
     @Autowired
     private IPhotoService photoService;
+    private ModelAndView modelAndView = new ModelAndView();
```

Comment:
There could be good reason for keeping this scoped locally to the methods.  If attribute scope, as it is here, then you could set a property in one method, and then that property re-appears in other methods, unless it is reset in those methods.  I'd lean towards leaving this as it were.